package.json

```diff
@@ -187,6 +188,7 @@
     "@types/react-test-renderer": "^18.0.0",
     "@types/sprintf-js": "^1.1.2",
     "@types/url-parse": "^1.4.8",
+    "@types/uuid": "^7.0.0",
```

Comment:
no yarn.lock change

Reply:
This version is already in yarn.lock, so no change needed.